ouroboros-consensus/src/Ouroboros/Consensus/Node/Run.hs

```diff
@@ -71,6 +71,8 @@ class ( Typeable blk
       , SerialiseNodeToClient blk blk
       , SerialiseNodeToClient blk (Serialised blk)
       , SerialiseNodeToClient blk (GenTx blk)
+      , SerialiseNodeToClient blk (GenTxId blk)
```

Comment:
These new constraints suggest that we need to add two such test to `goldenTest_SerialiseNodeToClient` in the `ouroboros-consensus-test/src/Test/Util/Serialisation/Golden.hs` file. I expect that to be a trivial diff 🤞

Reply:
I'll have a look tomorrow and re-rebase the branch since there are new conflicts as it seems 🤔, things are moving fast around here!

Reply:
Done in: feff0a618